src/main/java/com/maria/library/facade/mapper/AuthorMapper.java

```diff
@@ -0,0 +1,16 @@
+package com.maria.library.facade.mapper;
+
+import com.maria.library.facade.dto.AuthorDto;
+import com.maria.library.facade.dto.RegisterDto;
+import com.maria.library.model.Author;
+import com.maria.library.model.UserData;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AuthorMapper {
+
+    public Author mapToEntity(AuthorDto authorDto) {
+        Author authorData   = new Author(authorDto.getFirstname(), authorDto.getLastname(), authorDto.getNumberOfBooks());
```

Comment:
actualy, I don t think you need to instantiate an object and then return that object. You can transform this method into a lamda method and return directly the new instantiated object.